lib/SILGen/SILGenExpr.cpp

```diff
@@ -1894,12 +1908,25 @@ RValue RValueEmitter::visitEnumIsCaseExpr(EnumIsCaseExpr *E,
                                       {{E->getEnumElement(), t}});
   }
   
-  // Call the _getBool library intrinsic.
+  // Call the Bool(_builtinBooleanLiteral:) initializer
+  DeclName name(ctx, DeclBaseName::createConstructor(),
+                { ctx.Id_builtinBooleanLiteral });
+  auto members = ctx.getBoolDecl()->lookupDirect(name);
```

Comment:
You might want to cache the result of this lookup in the ASTContext

Reply:
Can I cache this in [CSApply](https://github.com/apple/swift/pull/20948/files#diff-cf676abf2776cbcdb21380661ec41563R3253)?